handler.test.js

```diff
@@ -10,7 +10,7 @@ const context = {
 }
 
 describe('Handler', () => {
-  it('Parses the event data into JSON', () => {
+  it('Parses the JSON body into a usable object', () => {
```

Comment:
Maybe 'Parses the JSON string into a JavaScript object'? I have to say I thought JSON was the object already so I'd be confused either way